### ISSUE TITLE
replicating image-level classification

### ISSUE THREAD

**Issue by rutescher** [tgxs002/wikiscenes#2]
Thank you for your work, the paper was a great read.

I was hoping to try the whole image classification task (as seen in table 1 in the paper) with different data. I.e., I would like to evaluate the model on data with ground truth image labels, but not ground truth masks.
`launch/run_wikiscenes_resnet50_cls.sh` and `launch/run_wikiscenes_mobilenetv2_cls.sh` seem to be the appropriate training scripts.

However, I haven't been able to find any code for the appropriate eval. 
Did I miss it or would I need to implement this myself? 
Any hints would be much appreciated.

**Comment by tgxs002** [comment]
Hi Rutescher,

Thank you for your attention on our work!

The evaluation code is here:

https://github.com/tgxs002/wikiscenes/blob/22b7b1de81dfe5fa50e25914d34f32ac3ff05447/train.py#L768

Hope it helps!

**Comment by tgxs002** [comment]
Also, the classification accuracy is periodically reported in the TensorBoard. You may find it useful.
